schema.sql

```diff
@@ -3,10 +3,10 @@
 CREATE TABLE dbo.animal
 (
     id INT NOT NULL PRIMARY KEY, -- primary key column
-    name [NVARCHAR](50) NOT NULL,
-    date_of_birth [NVARCHAR](50) NOT NULL,
-    escape_attempts [NVARCHAR](50) NOT NULL,
-    neutered [NVARCHAR](50) NOT NULL,
-    weight_kg [NVARCHAR](50) NOT NULL
+    name [NVARCHAR](100) NOT NULL,
```

Comment:
You corrected all data types, but not the `name` on line 6.

According to the requirements:

![image](https://user-images.githubusercontent.com/57421823/155483279-4f3c026e-ba01-4732-bbd7-1321971cde0b.png)

Therefore:

- [ ] As stated by the previous reviewer, please correct your `name` data type.

## Explanation:

NVARCHAR is used only by Oracle/Sybase/MSSQL.   We are using PostgreSQL.

## Hints:

If you want to constraint the number of characters, in PostgreSQL we use `VARCHAR(number of characters allowed)`.

So, in your implementation with a limit of 100 characters you could just use:

`name varchar(100) NOT NULL,`  

...or if you want to allow any number of characters:

`name string NOT NULL,`

Now that you know how to do it, please, kindly implement the change to correct the name column data type. 💪